mythril/laser/smt/solver/solver.py

```diff
@@ -45,14 +45,14 @@ def append(self, *constraints: List[Bool]) -> None:
         self.add(*constraints)
 
     @stat_smt_query
-    def check(self) -> z3.CheckSatResult:
+    def check(self, *args) -> z3.CheckSatResult:
```

Comment:
This is for using assumptions assumptions, it's obsolete now but we can keep it for future if needed